packages/react/runtime/types/types.d.ts

```diff
@@ -260,7 +260,11 @@ declare module '@lynx-js/types/background' {
 
   interface Performance {
     _generatePipelineOptions?(): PipelineOptions;
-    _onPipelineStart?(pipelineID: string, options: PipelineOptions): void;
+    /**
+     * @param pipelineID Returned by native when calling `_generatePipelineOptions`
+     * @param options since 3.1. PipelineOptions.
```

Comment:
The format of this JSDoc is not correct.

```suggestion
     * @param options - Added since 3.1. See {@link PipelineOptions} for more details.
```

Reply:
Done